x/programs/v2/runtime/config.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
 // See the file LICENSE for licensing terms.
 
-package v2
+package runtime
```

Comment:
🤔  I wouldn't have expected the rename to happen in your PR... Wouldn't it just be better to get rid of the whole v2 package/folder in one shot where that's all that's happening in the PR?

Reply:
Yeah, I'll probably do that in a separate PR after https://github.com/ava-labs/hypersdk/pull/917

Reply:
@dboehm-avalabs I think that you solved this by moving stuff from `v2` to `runtime` ?